docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -1,38 +1,31 @@
 = Migrating Ceph Manager daemons to {Ceph} nodes
 
-The following section describes how to move Ceph Manager daemons from the
-{rhos_prev_long} Controller nodes to a set of target nodes. Target nodes might
-be pre-existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is
-deployed by {OpenStackPreviousInstaller} with an HCI topology.
-This procedure assumes that Cephadm and the {Ceph} Orchestrator are the tools
-that drive the Ceph Manager migration. As is done with the other Ceph daemons
-(MDS, Monitoring, and RGW), the procedure uses the Ceph spec to modify the
-placement and reschedule the daemons. Ceph Manager is run in an active/passive
-fashion, and it also provides many modules, including the Ceph orchestrator.
+You must migrate your Ceph Manager daemons from the {rhos_prev_long} ({OpenStackShort}) Controller nodes to a set of target nodes. Target nodes are either existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is deployed by {OpenStackPreviousInstaller} with a Hyperconverged Infrastructure (HCI) topology.
 
+The following procedure uses `cephadm` and the Ceph Orchestrator to drive the Ceph Manager migration, and the Ceph spec to modify the placement and reschedule the Ceph Manager daemons. Ceph Manager is run in an active/passive state, and it also provides many modules, including the Ceph Orchestrator.
 
 .Prerequisites
 
-* Configure the target nodes (CephStorage or ComputeHCI) to have both `storage`
-and `storage_mgmt` networks to ensure that you can use both {Ceph} public and
-cluster networks from the same node. This step requires you to interact with
-{OpenStackPreviousInstaller}. From {rhos_prev_long} {rhos_prev_ver} and later
-you do not have to run a stack update.
+* The target nodes, CephStorage or ComputeHCI, are configured to have both `storage` and `storage_mgmt` networks. This ensures that you can use both {Ceph} public and cluster networks from the same node.
++
+[NOTE]
+This step requires you to interact with {OpenStackPreviousInstaller}. From {OpenStackShort} {rhos_prev_ver} and later you do not have to run a stack update.
 
 .Procedure
 
-. Ssh into the target node and enable the firewall rules that are required to
-  reach a Manager service:
+. SSH into the target node and enable the firewall rules that are required to reach a Ceph Manager service:
 +
 ----
 dports="6800:7300"
 ssh heat-admin@<target_node> sudo iptables -I INPUT \
     -p tcp --match multiport --dports $dports -j ACCEPT;
 ----
 +
-Repeat this step for each `<target_node>`.
+* Replace `<target_node>` with the name of the node you are migrating to.
```

Comment:
Should the following text match the text in step 3?
Change
"Replace `<target_node>` with the name of the node you are migrating to."
to
" Replace `<target_node>` with the hostname of the hosts listed in the {Ceph}
  through the `ceph orch host ls` command." ?